README.md

```diff
@@ -12,25 +12,107 @@ gem 'makwa'
 
 And then execute:
 
-    $ bundle install
+```shell
+$ bundle install
+```
 
 Or install it yourself as:
 
-    $ gem install makwa
+```shell
+$ gem install makwa
+```
+
+Makwa extends on the Active Interactions gem, if you do not have this installed, you can add it to your Gemfile:
+
```

Comment:
Manual inclusion/installation of active_interaction is not necessary, can be removed.